### DISCUSSION TITLE
Ender 3V2 / Sprite Pro / CR-Touch - Repetier Server

### DISCUSSION THREAD

**Discussion by kadi1991** [Jyers/Marlin#2072] [General Discussion]
hello together

i have converted my ender 3v2 to the sprite pro extruder with cr-touch. i have also tried various firmware. however, my problem is that my print head always moves to the right against the stop on the x-axis during levelling. i think that this is due to the alignment/definition of the print bed or various offset settings or Y/X-Max/Min settings. however, i could not find anything on the Jyers about how i can adjust these settings. 

anyone here who has/had the same problem?

thank you very much!

**Comment by mriscoc** [comment]
Check your X axis wiring, they could be inverted.
